packages/compiler/test/formatter/formatter.test.ts

```diff
@@ -1384,6 +1384,25 @@ model Bar {
       });
     });
 
+    it("keeps comment between decorators and model property", async () => {
```

Comment:
added those test for model property and enum member to check if they were working(which they were) but keeping now